### DISCUSSION TITLE
Increasing the minimum validator commission

### DISCUSSION THREAD

**Discussion by ruslan-tsybenov** [akash-network/community#265] [Governance Proposals]
### Overview
We are proposing the idea of increasing the minimum validator commission rate to 5%. 
It would have a positive influence on the stability of the network and Akash ecosystem.
### Problem
When validators offer 0% commission rates, it creates a situation where competition revolves around reducing commissions rather than providing validation services. 
### Solution
Introducing a minimum commission rate of 5% promotes healthy competition and encourages validators to focus on providing better services.
Earning a reasonable commission enables validators to manage their operational costs, make infrastructure upgrades, and support the security of the network.
### Conclusion
We appreciate the opinion of other community members about this proposal.
The collaborative decision is crucial for the success of the network.

**Comment by zJuuu** [comment]
There was a gov proposal (26) about the same topic last year that passed.
This decision should be enforced.

**Comment by troian** [comment]
it is coming in [v0.24.0](https://github.com/akash-network/support/issues/73) upgrade
